src/library/serato/seratofeature.cpp

```diff
@@ -1059,9 +1062,15 @@ void SeratoFeature::activateChild(const QModelIndex& index) {
 }
 
 void SeratoFeature::onSeratoDatabasesFound() {
-    QList<TreeItem*> foundDatabases = m_databasesFuture.result();
-    TreeItem* root = m_childModel.getRootItem();
+    std::vector<std::unique_ptr<TreeItem>> foundDatabases;
+    const QList<TreeItem*> result = m_databasesFuture.result();
+    for (const auto& pDatabaseFound : result) {
+        foundDatabases.emplace_back(pDatabaseFound);
+    }
```

Comment:
Does iterator-based construction also work?
```suggestion
    const QList<TreeItem*> result = m_databasesFuture.result();
    std::vector<std::unique_ptr<TreeItem>> foundDatabases(result.begin(), result.end());
```